src/panel/disconnected/Disconnected.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from "react";
+import styled, { createGlobalStyle } from "styled-components";
+import image from "../../assets/icon.svg";
+
+export const Disconnected: FC = () => (
+  <>
+    <GlobalStyle />
+    <Container>
+      <Logo alt={"Urql Eagle"} src={image} />
```

Comment:
Just a tiny nit but this doesn't really need an `alt` since it's not really giving any information/context to a screen reader user

Reply:
Gotta disagree with this one. Regardless of whether the image is important or not, it's valuable to the user to know what they're looking at.

Reply:
In general logos like these are considered decorative and are usually supplied an empty `alt` text so the screen reader will know to skip it. 

https://www.w3.org/WAI/tutorials/images/decorative/